### PR TITLE
HAS: Manage image refs in this repo & controller manifests from source repo

### PR DESCRIPTION
**Pull request by sbose78**
- [x]  Removed OLM manifests
- [x] used a remote 'base' to consume manifests from https://github.com/redhat-appstudio/application-service/config/default
- [x]  replaced image reference to latest commit in the [component ](https://github.com/redhat-appstudio/application-service/pull/23). Needs another commit after the PR for aggregation of roles is merged.
- [x]  grant Argo CD permissions to CRUD CRDs
- [x]  aggregation of roles https://github.com/redhat-appstudio/application-service/pull/23 To test this, use the line 
```
# - https://github.com/redhat-appstudio/application-service/config/default?ref=0f7b9c8fcc7d6422022a51a0711b3e0c38777450 
```
in kustomization.yaml

### CODE REVIEW

File: components/has/kustomization.yaml [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 
 apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
\ No newline at end of file
+kind: Kustomization
+
+images:
+- name: quay.io/redhat-appstudio/application-service
+  newName: quay.io/redhat-appstudio/application-service
+  newTag: sha-9b9dcc9
```
Comment (sbose78) [inline components/has/kustomization.yaml:13]: This is what will be bumped again and again.

Comment (johnmcollier) [reply components/has/kustomization.yaml:13]: Can (and/or should?) we point to a specific ref in https://github.com/redhat-appstudio/application-service too?

File: components/has/kustomization.yaml [hunk 2]
```diff
@@ -1,10 +1,15 @@
 resources:
 - allow-argocd-to-manage.yaml
-- catalogsource.yaml
-- operatorgroup.yaml
-- subscription.yaml
+- https://github.com/redhat-appstudio/application-service/config/default
```
Comment (johnmcollier) [inline components/has/kustomization.yaml:3]: Can (or is it possible to?) we point to a specific ref in https://github.com/redhat-appstudio/application-service too?

Comment (sbose78) [reply components/has/kustomization.yaml:3]: Yes.
Like this
github.com/Liujingfang1/kustomize/examples/helloWorld?ref=repoUrl2

Comment (sbose78) [reply components/has/kustomization.yaml]: - https://github.com/redhat-appstudio/application-service/config/default?ref=e97f8514f0552df4bb92d4bc7133e6991f77af97

**Review by johnmcollier — APPROVED** [review]
Looks good from my end. No concerns from this approach 👍

**Review by johnmcollier — APPROVED** [review]

**Comment by sbose78** [comment]
Took in the latest commit from, need another round of review @johnmcollier, please

https://github.com/redhat-appstudio/application-service/commits/main

**Review by johnmcollier — APPROVED** [review]

**Review by johnmcollier — APPROVED** [review]

**Pull request merged**